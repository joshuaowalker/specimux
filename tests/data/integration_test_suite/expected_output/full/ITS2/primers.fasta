>gITS7 position=forward pool=ITS2
GTGARTCATCGARTCTTTG
>ITS4 position=reverse pool=ITS,ITS2
TCCTCCGCTTATTGATATGC
