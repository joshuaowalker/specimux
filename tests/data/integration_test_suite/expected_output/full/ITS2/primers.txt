>gITS7
GTGARTCATCGARTCTTTG
>ITS4
TCCTCCGCTTATTGATATGC
